### DISCUSSION TITLE
[License]: Is it possible to use grafana for my commercial data lakehouse platform for disconnected environment.

### DISCUSSION THREAD

**Discussion by cloudcheflabs** [grafana/grafana#70213] [General]
I want to use grafana without modification of grafana sources as monitoring tool with prometheus for my commercial data lakehouse platform for disconnected environment. 
Is there any license problem to use grafana without source codes modification for my commercial data lakehouse platform?

- Kidong.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
